qemu/qmp.go

```diff
@@ -1023,13 +1033,13 @@ func (q *QMP) ExecuteNetPCIDeviceAdd(ctx context.Context, netdevID, devID, macAd
 // using the device_add command. devID is the id of the device to add.
 // Must be valid QMP identifier. netdevID is the id of nic added by previous netdev_add.
 // queues is the number of queues of a nic.
-func (q *QMP) ExecuteNetCCWDeviceAdd(ctx context.Context, netdevID, devID, macAddr, addr, bus string, queues int) error {
+func (q *QMP) ExecuteNetCCWDeviceAdd(ctx context.Context, netdevID, devID, macAddr, bus string, queues int) error {
 	args := map[string]interface{}{
 		"id":     devID,
 		"driver": VirtioNetCCW,
 		"netdev": netdevID,
 		"mac":    macAddr,
-		"addr":   addr,
+		"devno":  bus,
```

Comment:
What was addr used for previously?  Was this only used by s390?

Reply:
See the previous comment. It was not used

Reply:
I see and this is a s390 specific function so I guess we should be okay.